src/supervisor/worker.js

```diff
@@ -73,7 +74,7 @@ function main () {
 
     // Require the target module to load the function for invocation
     const functionModule = require(localdir);
-    const handler = functionModule[cloudfunction.entryPoint || name];
+    const handler = _.get(functionModule, cloudfunction.entryPoint, functionModule[name]);
```

Comment:
Actually, this would be cleaner as:
`const handler = _.get(functionModule, cloudfunction.entryPoint || name);`